bin/main.rb

```diff
@@ -1,34 +1,74 @@
 #!/usr/bin/env ruby
 
-puts "-------------Welcome to Tic Tak Toe-----------------"
-print "Enter Player 1 name: "
+require_relative '../lib/game_logic.rb'
+require_relative '../lib/player.rb'
+require_relative '../lib/board.rb'
 
-player_1 = gets.chomp.capitalize
+puts '-------------Welcome to Tic Tak Toe-----------------'
+print 'Enter Player 1 name: '
```

Comment:
You should create a validation to avoid the player's name be left empty (would be nice if it couldn't be duplicated, too)